RoaringBitmap/src/main/java/org/roaringbitmap/RoaringArray.java

```diff
@@ -998,6 +1022,17 @@ public int last() {
     return lastKey << 16 | container.last();
   }
 
+  /**
+   * Closes the containers used and frees the arrays used by them
+   */
+  public void close() {
```

Comment:
Freeing the memory seems to depend on this method being called. There is a **lot** of code out there which will need to be migrated to ensure the memory is freed. This is a big red flag.

Reply:
At least the default implementation doesn’t retain references, so this won’t cause memory leaks in existing applications.

Reply:
Yes, the idea is that I didn’t change the public API and existing users won’t need to change anything as long as they don’t benefit from the custom memory management